src/garage/torch/policies/context_conditioned_policy.py

```diff
@@ -0,0 +1,219 @@
+# pylint: disable=attribute-defined-outside-init
```

Comment:
As far as the code is concerned, I think this file looks good. I haven't checked the math though, since I'm not familiar with PEARL.